### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by mvrueden**

### CODE REVIEW

File: src/app/model/Player.ts [hunk 1]
```diff
@@ -99,4 +101,16 @@ export class Player {
     }
     return resourceSites[0];
   }
+
+  setNotifications(notifications: Notification[]) {
+	this.notifications = notifications;
```
Comment (mvrueden) [inline src/app/model/Player.ts]: formatting issue

File: src/app/model/Player.ts [hunk 2]
```diff
@@ -99,4 +101,16 @@ export class Player {
...
+  }
+
+  getNotifications() : Notification[] {
+	return this.notifications;
```
Comment (mvrueden) [inline src/app/model/Player.ts]: formatting issue

File: src/app/model/Player.ts [hunk 3]
```diff
@@ -99,4 +101,16 @@ export class Player {
...
+  }
+
+  getUnreadNotifications() : Notification[] {
+	return this.notifications.filter(notification => !notification.read);
```
Comment (mvrueden) [inline src/app/model/Player.ts]: formatting issue

File: src/app/modules/app-routing.module.ts [hunk 4]
```diff
@@ -16,6 +18,8 @@ const routes: Routes = [
       { path: 'overview', component: OverviewComponent, canActivate: [AuthGuard], data: { title: 'Overview' }},
       { path: 'resources', component: ResourcesComponent, canActivate: [AuthGuard], data: { title: 'Resource List' }},
       { path: 'buildings', component: BuildingsComponent, canActivate: [AuthGuard], data: { title: 'Building List' }},
+	  { path: 'messages', component: MessageOverviewComponent, canActivate: [AuthGuard], data: { title: 'Building List' }},
```
Comment (mvrueden) [inline src/app/modules/app-routing.module.ts]: formatting issue

File: src/app/modules/sidebar/sidebar.component.html [hunk 5]
```diff
@@ -12,10 +12,12 @@
         <div class="user-info">
           <span class="user-name">{{player.getName()}}
           </span>
+<!--
```
Comment (mvrueden) [inline src/app/modules/sidebar/sidebar.component.html]: Pleaser remove completely if it needs to be removed (-:

File: src/app/services/player.service.ts [hunk 6]
```diff
@@ -74,6 +75,11 @@ export class PlayerService {
       return new ResourceSite(site.id, site.storage.resource, site.storage.amount, site.storage.capacity, site.droneCount);
     });
     player.setResourceSites(sites);
+
+	const notifications = playerJson.notifications.map(notification => {
+		return new Notification(1, notification.label, notification.content, notification.read, notification.completionDate);
```
Comment (mvrueden) [inline src/app/services/player.service.ts]: Notifications should get a dynamic id

File: src/app/model/Player.ts (hunk 1 above)
Comment (cjschimeczek) [reply src/app/model/Player.ts]: Replaced tab with white space

File: src/app/model/Player.ts (hunk 2 above)
Comment (cjschimeczek) [reply src/app/model/Player.ts]: Replaced tab with white space

File: src/app/modules/app-routing.module.ts (hunk 4 above)
Comment (cjschimeczek) [reply src/app/modules/app-routing.module.ts]: Replaced tab with white space

File: src/app/modules/sidebar/sidebar.component.html (hunk 5 above)
Comment (cjschimeczek) [reply src/app/modules/sidebar/sidebar.component.html]: deleted

File: src/app/services/player.service.ts (hunk 6 above)
Comment (cjschimeczek) [reply src/app/services/player.service.ts]: implemented

**Pull request merged**